README.md

```diff
@@ -39,7 +39,7 @@ It is currently possible to use CadQuery for your own projects in 3 different wa
 * From a [Jupyter notebook](https://github.com/bernhard-42/jupyter-cadquery)
 * As a standalone library
 
-The easiest way to install CadQuery and its dependencies is using conda:
+The easiest way to install CadQuery and its dependencies is using Anaconda:
```

Comment:
Why do you mention Anaconda here?

Reply:
We've got quite a few users who have never used Anaconda and it isn't mentioned until the Standalone section below this section. It gives users unfamiliar a cue to search for "Anaconda" if they don't know what it is and/or have never used it. It would probably be more clear if I change it to "using conda, which is included as part of an Anaconda/Miniconda installation".

Reply:
I've updated code README for this, please have a look when you get a chance.